.github/workflows/docker.yml

```diff
@@ -38,13 +38,14 @@ jobs:
       - name: Log in to Docker Hub
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: docker.pkg.github.com
```

Comment:
This should be `ghcr.io` - the `docker.pkg.github.com` is legacy

Reply:
More info: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry